server/protocols/jmap-rfc-8621-integration-tests/memory-jmap-rfc-8621-integration-tests/src/test/resources/jmap.properties

```diff
@@ -1,3 +1,5 @@
 # Configuration urlPrefix for JMAP routes.
-url.prefix=http://domain.com
+protocol.jmap=http
+protocol.ws=ws
+url.prefix=domain.com
```

Comment:
That is a breaking change. And breaking changes are bad.

I would propose:

```
url.prefix=http://domain.com
websocket.url.prefix=ws://domain.com
```

That way if you don't change your config, you will not end up with an unwanted behaviour.

Bonus: One could set up proxying to a dedicated websocket server, and advertise that through the session, using a completely distinct load balancer... <3